src/miral/launch_app.cpp

```diff
@@ -80,7 +80,8 @@ auto miral::launch_app(
 
         execvp(exec_args[0], const_cast<char*const*>(exec_args.data()));
 
-        throw std::logic_error(std::string("Failed to execute client (") + exec_args[0] + ") error: " + strerror(errno));
+        mir::log_warning("Failed to execute client (\"%s\") error: %s", exec_args[0], strerror(errno));
+        exit(EXIT_FAILURE);
```

Comment:
Should this be `mir::fatal_error`?

Reply:
No.

1 That _might_ be replaced by the shell to throw an exception; and, 
2. Logging it as "fatal" would be confusing as the original  Mir process continues perfectly happily.